jwst/assign_wcs/niriss.py

```diff
@@ -88,6 +89,28 @@ def niriss_soss_set_input(model, order_number):
     return wcs.WCS(pipeline)
 
 
+def _niriss_order_bounding_box(input_model, order):
+    import numpy as np
+    bbox_y = np.array([-0.5, input_model.meta.subarray.ysize - 0.5])
+    bbox_x = np.array([-0.5, input_model.meta.subarray.xsize - 0.5])
+
+    if order == 1:
+        return (tuple(bbox_y), tuple(bbox_x))
+    elif order == 2:
+        return (tuple(2 * bbox_y), tuple(2 * bbox_x))
+    elif order == 3:
+        return (tuple(3 * bbox_y), tuple(3 * bbox_x))
```

Comment:
Currently all orders have the same bbox, defined correctly under `order=1`. In the future we may want to change this but for now we'll use the entire subarray.

Reply:
See changes in 444742a.